articles/customization-using-bulma/index.md

```diff
@@ -0,0 +1,234 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/customization-using-bulma/
+title: Customize CSS using Bulma
+description: Bulma is a lightweight, open source CSS framework that has excellent customization features which is perfect for rapid development.
+author: saiharsha-balasubramaniam
+date: 2020-09-04T00:00:00-08:00
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/customization-using-bulma/hero.jpg
+    alt: bulma
+---
+
+**Cascading Style Sheets** is a stylesheet language that is used to describe how a webpage looks and feels visually. To speed up the development of projects, developers use CSS frameworks like Bulma. Let us see why CSS frameworks are useful and learn about Bulma.
+
+## Table of Contents
+
+- [The Need for a CSS Framework](#need-for-a-css-framework)
+- [Bulma](#bulma)
+- [Getting Started](#getting-started)
+- [Column System](#column-system)
+- [Components](#components)
+- [Customizing Variables](#customizing-variables)
+- [Further Reading](#further-reading)
+
+## The Need for a CSS Framework
+
+CSS is an flexible language, with a ton of customization options. This may sometimes be overwhelming for a beginner. When you're out to build a product, a CSS framework helps to accelerate the process of development. It helps to reduce time to production, which is crucial for a project. Some other benefits to using a CSS framework include:
+
+- Good CSS frameworks have excellent documentation.
+- Easier to make websites responsive.
+- Cross-browser support.
+
+## Bulma
+
+Bulma is a fast, lightweight and customizable CSS framework. Bulma is perfect for production-grade web applications due to its excellent performance and small size.
+
+- Bulma is based on CSS Flexbox.
+- It has an excellent developer community.
+- It can be customized extensively through variables.
+- Bulma uses easy-to-read class names that are easy to remember.
+- It provides a modern and clean design.
+
+For the above reasons, Bulma is an efficient CSS framework that can be used to create performant websites.
+
+## Getting Started
+
+Let's add Bulma to our web application. We can do this by one of the following ways.
+
+- We can use the Bulma CDN to add Bulma to our webpage. To do so in the website's `<head>`, add the following line.
+
+```html
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.0/css/bulma.min.css"/>
+```
+
+- Download the npm package from the npm repository.
+
+```bash
+npm install bulma
+```
+
+- We can also download the `bulma.css` file from [here](https://github.com/jgthms/bulma/tree/master/css) and include it in our webpage using the `<link>` tag.
```

Comment:
That link is only to the CSS folder, not to the file that downloads it. I'd also recommend using the minified version. https://github.com/jgthms/bulma/blob/master/css/bulma.min.css